pkg/metrics/metrics.jsx

```diff
@@ -333,6 +358,32 @@ class CurrentMetrics extends React.Component {
                 });
     }
 
+    onTemperatureUpdate(event, message) {
+        debug("current CPU temperature  message", message);
+        const data = JSON.parse(message);
+
+        if (!Array.isArray(data)) {
+            return;
+        }
+
+        decompress_samples(data, this.temperatureSamples);
+        this.cpuTemperature = parseInt(this.temperatureSamples[0]);
+
+        if (this.cpuTemperature <= 80) {
+            this.cpuTemperatureColors.textColor = "";
+            this.cpuTemperatureColors.iconColor = "";
+            this.cpuTemperatureColors.icon = null;
+        } else if (this.cpuTemperature < 95) {
+            this.cpuTemperatureColors.textColor = "text-color-warning";
+            this.cpuTemperatureColors.iconColor = "icon-color-warning";
+            this.cpuTemperatureColors.icon = <ExclamationTriangleIcon className="fa-exclamation-triangle" />;
```

Comment:
These class names are confusing. `fa-*` were the old-style FontAwesome icon names. But this has nothing to do with the icon shape, I suppose it's just for the tests to validate the current warning level?  Could you name these `temperature-{warning,critical}`?